src/main/resources/application.yaml

```diff
@@ -113,6 +113,9 @@ spring:
             loggerLevel: ${FEIGN_LOG_LEVEL:none}
 
 darts:
+  manual-deletion:
```

Comment:
i'm wondering if we should introduce an `automated-task` element to `darts` and then have `manual-deletion` within that?